src/snowflake/snowpark/mock/_functions.py

```diff
@@ -929,3 +930,99 @@ def mock_to_variant(expr: ColumnEmulator):
     res = expr.copy()
     res.sf_type = ColumnType(VariantType(), expr.sf_type.nullable)
     return res
+
+
+DATETIME_PART_TO_ALIASES = {
```

Comment:
is this documented somewhere in the snowflake official doc? if so can we have a link here for future reference